src/main/java/com/invoiced/entity/Subscription.java

```diff
@@ -196,7 +175,7 @@ public void cancel() throws EntityException {
 
 	public SubscriptionPreview preview() throws EntityException {
 
-		String url = this.getConnection().baseUrl() + "/" + this.getEntityName() + "/preview";
+		String url = this.getEndpoint(false, true) + "/preview";
```

Comment:
I could make this change, although the result would be more verbose than it is currently because the base URL has to be fetched regardless. I don't see a downside to keeping it how it is. Thoughts?